cms/djangoapps/models/settings/course_details.py

```diff
@@ -37,16 +37,16 @@ class CourseDetails(object):
     def __init__(self, org, course_id, run):
         # still need these for now b/c the client's screen shows these 3 fields
         self.org = org
-        self.course_id = course_id # This actually holds the course number.
+        self.course_id = course_id  # This actually holds the course number.
         self.run = run
         self.start_date = None  # 'start'
         self.end_date = None  # 'end'
         self.enrollment_start = None
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
-        self.short_description = ""
-        self.overview = ""  # html to render as the overview
-        self.about_sidebar_html = "" # html to render as the about_sidebar_html
+        self.short_description = ''
```

Comment:
It's important to group together logical changesets and conversely, exclude non-related changes.

This line didn't have any PEP8 violations, so we shouldn't be editing it at this time. I appreciate that you opted for consistency, though doing more than one thing at a time (here a stylistic change vs. the PEP8 changes elsewhere) makes it difficult to follow the `git log` history.

In cases where we already need to edit a line, we can make sure that the line follows all of our requirements, but we shouldn't touch unaffected code.